src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx

```diff
@@ -20,6 +20,7 @@ redirects:
   - /docs/using-new-relic/
   - /docs/using-new-relic/welcome-new-relic
   - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
+  - docs.newrelic.com/docs/new-relic-solutions/new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020
```

Comment:
Should this start with the /docs/ portion?

Reply:
done! thanks!